code/style.css

```diff
@@ -1,7 +1,322 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body { 
+  margin: 0.1px;
+  }
+
+header{
+    display: flex;
+    flex-direction: column;
+    justify-content:start;
+    padding-bottom: 20em;
+    text-align: center;
+  }
+
+.hero-title{
+    font-family: 'Forum', cursive;
+    letter-spacing: 10px;
+    text-align: center;
+    margin-bottom: 10em;
+    color: white;
+
+  }
+  /* This is the text box ontop of the header video */
+
+.hero-text-box {
+  width: 90%;
+  margin-left: 4em;
+  max-width: 800px;
+}
+.h2 {
+  color: whitesmoke;
+  font-family: 'Work Sans', sans-serif;
+  text-align: left;
+  font-size: 40px;
+  width: 60%;
+  padding-left: 1.75em;
+  letter-spacing: 0.10em;
+}
+
+.header-text{
+  color: whitesmoke;
+  font-family: 'Work Sans', sans-serif;
+  text-align: left;
+  width: 60%;
+  line-height: 1.5em;
+  padding-left: 3em;
+  font-size: 20px;
+}
```

Comment:
The contrast of the texts are not so good when they are on top of the spinning globe. Maybe put a translucent backgorund just so users can read the text easily.

Reply:
I will check out a translucent background, sounds interesting.